app/controllers/floating_ip_controller.rb

```diff
@@ -14,4 +14,260 @@ def self.display_methods
   end
 
   menu_section :net
+
+  def button
```

Comment:
Can you cleanup this method a bit?

Please, make it a simple case statement and make sure that all the code is needed. E.g. you are setting the `@refresh_div` variable and then testing it. Is there some code between the statements that might change that value?